rest/role_api_test.go

```diff
@@ -350,7 +350,7 @@ func TestRoleAccessChanges(t *testing.T) {
 		channels.TimedSet{
 			"!":    channels.NewVbSimpleSequence(1),
 			"beta": channels.NewVbSimpleSequence(zegpold.Sequence()),
-		}, zegpold.InheritedChannels())
+		}, zegpold.InheritedCollectionChannels(base.DefaultScope, base.DefaultCollection))
```

Comment:
is it going to be more correct to do `rt.GetSingleTestCollection().ScopeName()` and `rt.GetSingleTestCollection.CollectionName()` for future compatibility with non test collection? We can leave this for Greg's future task.

Reply:
Yes - I started to make that change to use NewRestTester instead of NewRestTesterDefaultCollection, and then use SingleTestCollection for scope and collection name, but it also required the changes to the user creation to assign collection-scoped channels e.g.:
```
response := rt.SendAdminRequest("PUT", "/db/_user/alice", `{"password":"letmein", "admin_channels":["alpha"]}`)
```
so I decided to leave it for future work.